help/configure-notification-bot.md

```diff
@@ -69,6 +69,27 @@ these messages is "signups".
 
 {end_tabs}
 
+### Zulip update announcements
+
+In Zulip, you can get notified about significant changes
+to the Zulip user experience such as new features, new keyboard shortcuts,
+changes in terminology, etc.
```

Comment:
I think we want to tweak this somehow to make it clear that this is an org-wide configuration, not a personal one. Something like this:

> Your organization can get notified about significant changes
in Zulip, such as new features, new keyboard shortcuts, etc.

Reply:
@timabbott might have further suggestions.

Reply:
Maybe something like 

> Zulip announces new features and important product and design changes via automated notifications sent to the "Zulip updates" topic in a configurable stream.